src/App.js

```diff
@@ -1,20 +1,28 @@
 // import "./css/App.css";
-import React from 'react'
+import React, { useState } from 'react'
 import { useRecoilState } from 'recoil'
 import { testState } from './store/atom'
+import { ThemeProvider } from 'styled-components'
 //import components here:
 import NavBar from './components/Navbar/Navbar'
 import SignIn from './components/auth/SignIn'
 import SignUp from './components/auth/SignUp'
-import { Grommet } from 'grommet'
-import { grommet, dark } from 'grommet/themes'
+import darkTheme from './theme/darkTheme'
+import lightTheme from './theme/lightTheme'
 
 const App = () => {
+  const [isDarkTheme, setIsDarkTheme] = useState(false)
+
+  const toggleTheme = () => {
```

Comment:
I'm guessing you meant to set the theme selected into localStorage here? 
(In reference to the get from localStorage in /theme/index.js)